tests/unit/lms/views/api/gateway_test.py

```diff
@@ -45,12 +48,43 @@ def test_it_checks_for_guid_agreement(self, context, pyramid_request):
         with pytest.raises(HTTPForbidden):
             h_lti(context, pyramid_request)
 
+
+@pytest.mark.usefixtures("grant_token_service")
+class TestHLTIConsumer:
+    # These tests are "consumer tests" and ensure we meet the spec we have
+    # provided to our users in our documentation
+
+    def test_schema_is_valid(self, validator, schema):
+        validator.check_schema(schema)
+
+    def test_schema_examples_are_valid(self, validator, schema):
+        for example in schema["examples"]:
+            validator.validate(example)
+
+    def test_gateway_output_matches_the_schema(
+        self, validator, context, pyramid_request
+    ):
+        response = h_lti(context, pyramid_request)
+
+        validator.validate(response)
```

Comment:
Yup, this is not a lot of work but checking we are returning something that looks like what we are saying we return will go along way to keep the docs up to date.